pfcpiface/config.go

```diff
@@ -99,6 +100,7 @@ type P4rtcInfo struct {
 	P4rtcServer string          `json:"p4rtc_server"`
 	P4rtcPort   string          `json:"p4rtc_port"`
 	QFIToTC     map[uint8]uint8 `json:"qfi_tc_mapping"`
+	DefaultTc   uint8           `json:"default_tc"`
```

Comment:
nit: The Go convention is to use uppercase for acronyms, if possible. I suggest using `DefaultTC`